src/proxyvisor.coffee

```diff
@@ -48,6 +48,10 @@ router.get '/v1/devices', (req, res) ->
 		res.status(503).send(err?.message or err or 'Unknown error')
 
 router.post '/v1/devices', (req, res) ->
+	appId = req.body.appId
+	if !appId? or _.isNaN(parseInt(appId)) or parseInt(appId) <= 0
```

Comment:
validating a null or <=0 appId on the supervisor side is good, but should't this request resulted in a KO from the API?